pkg/tenant/api/v1/client.go

```diff
@@ -288,6 +289,105 @@ func (s *APIv1) MemberCreate(ctx context.Context, in *Member) (out *Member, err
 	return out, nil
 }
 
+func (s *APIv1) TenantProjectList(ctx context.Context, id string, size uint32, tok string, in PageQuery) (out *TenantProjectPage, err error) {
+	if id == "" {
+		return nil, ErrMemberIDRequired
```

Comment:
Should this be `ErrProjectIDRequired`?

Reply:
You're right. I need add an error for each resource type.